src/features/instantiate/containers/UnknownError.tsx

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
 
+import { Button } from '@digdir/design-system-react';
+
+import { DevToolsActions } from 'src/features/devtools/data/devToolsSlice';
+import { DevToolsTab } from 'src/features/devtools/data/types';
 import { InstantiationErrorPage } from 'src/features/instantiate/containers/InstantiationErrorPage';
 import { useLanguage } from 'src/hooks/useLanguage';
 
+const devHostNames = ['local.altinn.cloud', 'dev.altinn.studio', 'altinn.studio', 'studio.localhost', 'tt02.altinn.no'];
+const isDev = devHostNames.some((host) => window.location.hostname.endsWith(host));
```

Comment:
Isn't this duplicated from DevTools? Should probably make a `useIsDev()` hook instead

Reply:
Will do, only difference is that this includes tt02